FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/ContextualCommandBar.kt

```diff
@@ -0,0 +1,276 @@
+package com.microsoft.fluentui.contextualcommandbar
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.support.annotation.DrawableRes
+import android.support.v4.content.ContextCompat
+import android.support.v7.content.res.AppCompatResources
+import android.util.AttributeSet
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import android.widget.HorizontalScrollView
+import android.widget.ImageView
+import android.widget.LinearLayout
+import com.microsoft.fluentui.R
+import com.microsoft.fluentui.util.ThemeUtil
+import com.microsoft.fluentui.util.isVisible
+
+class ContextualCommandBar @JvmOverloads constructor(
```

Comment:
Just starting to look through the PR, and it is not clear to me how icon state is to be updated. Updating the CommandItem selected/enabled state will propagate to the UI representation through the theme bindings?